python/docs/source/getting_started/install.rst

```diff
@@ -19,71 +19,95 @@
 Installation
 ============
 
-Official releases are available from the `Apache Spark website <https://spark.apache.org/downloads.html>`_.
-Alternatively, you can install it via ``pip`` from PyPI.  PyPI installation is usually for standalone
-locally or as a client to connect to a cluster instead of setting a cluster up.  
+PySpark is included in the official releases of Spark available in the `Apache Spark website <https://spark.apache.org/downloads.html>`_.
+For Python users, PySpark also provides ``pip`` installation from PyPI. This is usually for standalone
+locally or as a client to connect to a cluster instead of setting a cluster itself up.
```

Comment:
standalone locally -> local usage?
"instead of setting up a cluster itself" maybe